cd/application-deployment/dev/vaec-celery-task-definition.json

```diff
@@ -16,218 +16,20 @@
           "awslogs-stream-prefix": "ecs"
         }
       },
-      "environment": [
-        {
-          "name": "ACCEPT_RECIPIENT_IDENTIFIERS_ENABLED",
-          "value": "True"
-        },
-        {
-          "name": "NIGHTLY_NOTIF_CSV_ENABLED",
-          "value": "True"
-        },
-        {
-          "name": "API_HOST_NAME",
-          "value": "https://dev.api.notifications.va.gov"
-        },
-        {
-          "name": "DD_ENV",
-          "value": "dev"
-        },
-        {
-          "name": "DD_SITE",
-          "value": "ddog-gov.com"
-        },
+      "environmentFiles": [
         {
-          "name": "DD_PROFILING_ENABLED",
-          "value": "True"
-        },
+          "type": "s3",
+          "value": "arn:aws-us-gov:s3:::vanotify-environment-variables-dev/dev.env"
+        }
+      ],
+      "environment": [
         {
           "name": "DD_SERVICE",
           "value": "celery"
         },
-        {
-          "name": "DD_PROFILING_ENABLE_CODE_PROVENANCE",
-          "value": "True"
-        },
-        {
-          "name": "API_MESSAGE_LIMIT_ENABLED",
-          "value": "True"
-        },
-        {
-          "name": "API_RATE_LIMIT_ENABLED",
-          "value": "True"
-        },
-        {
-          "name": "ATTACHMENTS_BUCKET",
-          "value": "dev-notifications-va-gov-attachments"
-        },
-        {
-          "name": "AWS_PINPOINT_APP_ID",
-          "value": "df55c01206b742d2946ef226410af94f"
-        },
-        {
-          "name": "AWS_SES_EMAIL_FROM_USER",
-          "value": "dev-do-not-reply"
-        },
-        {
-          "name": "CHECK_GITHUB_SCOPE_ENABLED",
-          "value": "True"
-        },
-        {
-          "name": "CHECK_TEMPLATE_NAME_EXISTS_ENABLED",
-          "value": "True"
-        },
-        {
-          "name": "EMAIL_ATTACHMENTS_ENABLED",
-          "value": "True"
-        },
-        {
-          "name": "EMAIL_PASSWORD_LOGIN_ENABLED",
-          "value": "True"
-        },
-        {
-          "name": "EMAIL_PROVIDER_SELECTION_STRATEGY_LABEL",
-          "value": "LOAD_BALANCING"
-        },
         {
```

Comment:
We are keeping this value in the task definition?

Reply:
There is a separate ticket about removing old or unused variables, this is just moving what's there to the environment files.